LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 TAB
```

Comment:
Will we need to change this to K+C at some point?

Reply:
I imagine so, yes